### DISCUSSION TITLE
Adding a whiteboard

### DISCUSSION THREAD

**Discussion by oumout** [st3v3nmw/obsidian-spaced-repetition#1046] [Q&A]
Hello,

Ankidroid has that option to enable whiteboard while you are working on a card. 

Before i see the answer i am able to write on screen and this makes me check my writing of foreign language. 

Is it possible to add a whiteboard option while answering cards? Ty.

**Comment by ronzulu** [comment]
Hi @oumout is this what you had in mind:
[ Text inputs for clozes #644 ](https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/644)

**Comment by oumout** [comment]
Hi,

 Yours is good solution for cloze cards but  what about basic cards.

My process is like that

1- See the card
2- Enable whiteboard
3-Write answer on screen with hand, touch pen etc..
4- Reveal the answer
5- Decide on your own if it is correct or not.

Using hand and it's gestures help us to memorize more i believe. 

It is a feature on ankidroid already with the same process i mentioned above. 

from ankidroid manual

"Enable / Disable Whiteboard

    This action enables or disables the whiteboard feature for the current deck. The whiteboard feature allows you to draw on the screen, which is particularly useful for practicing drawing characters from languages such as Japanese. When the whiteboard has been enabled for the current deck, two new actions will become available for clearing and hiding the whiteboard. Disabling the whiteboard will hide these actions as well as the whiteboard itself.
"

**Comment by ronzulu** [comment]
> Using hand and it's gestures help us to memorize more i believe.

Good point, makes sense that by having the recall process be more active that we might remember things better.

**Comment by LeonelRFF** [comment]
Hi, if at some point you plan to implement this, it would probably be better to do it with the excalidraw plugin, its creator in the future may implement a function (https://github.com/dai-shi/excalidraw-animate?tab=readme-ov-file#using-a-local-file) that makes this easier for Space repetition
